zerver/lib/webhooks/common.py

```diff
@@ -114,3 +133,25 @@ def validate_extract_webhook_http_header(request: HttpRequest, header: str,
         raise MissingHTTPEventHeader(header)
 
     return extracted_header
+
+
+def get_http_headers(integration_name: str, fixture_name: str) -> Dict["str", "str"]:
```

Comment:
get_fixture_http_headers` would be a clearer name for this, since its role is to get the HTTP headers given a fixture filename, and one could easily be confused into thinking this code runs in production.

Reply:
I see that you've already made this change before merging to master, so I'll mark this as resolved.